package.json

```diff
@@ -68,16 +73,25 @@
   "devDependencies": {
     "@telerik/kendo-common-tasks": "^3.0.0",
     "@telerik/semantic-prerelease": "^1.0.0",
+    "@types/jquery": "^2.0.48",
+    "@types/qunit": "^2.0.31",
     "bootstrap": "4.0.0-beta.2",
     "cz-conventional-changelog": "^1.1.5",
     "editorconfig-tools": "^0.1.1",
     "ghooks": "^1.0.3",
     "glob": "^7.0.5",
     "handlebars": "^4.0.10",
+    "jquery": "^1.9.1",
+    "karma": "^1.7.1",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-phantomjs-launcher": "^1.0.4",
+    "karma-qunit": "^1.2.1",
     "mime": "^1.3.4",
+    "qunitjs": "^2.4.1",
     "sass-lint": "^1.7.0",
     "sassdoc": "^2.1.20",
     "semantic-release": "^6.3.6",
+    "typescript": "^2.6.1",
```

Comment:
I don't see much justification for using TypeScript in the tests so far, apart from type completion for the `metrics` object. Perhaps we can shave off some of the dependencies and configuration boilerplate if we use JS?